src/components/npm-content/NpmContent.js

```diff
@@ -1,8 +1,22 @@
-import {Box, List, ListItem, ListItemText, Typography} from '@mui/material';
-import React from 'react';
+import {Box, Button, List, ListItem, ListItemText, Tooltip, Typography} from '@mui/material';
+import React, {useCallback, useState} from 'react';
 import {t} from '../../utils/translations';
+import Icon from '../icon';
+import copyIcon from '../../assets/icons/copy-blue.svg';
+import copyToClipboard from '../../utils/copyToClipboard';
 
 const NpmContent = ({version}) => {
+  const [showCopyTooltip, setShowCopyTooltip] = useState(false);
+  const command = `npm install --save op-payment-widget-v3@${version} --registry https://packagecloud.io/optile/javascript/npm/`;
+  const openTooltip = useCallback(() => {
+    setShowCopyTooltip(true);
+    setTimeout(() => setShowCopyTooltip(false), 2000);
+  }, []);
+  const handleOnClick = useCallback((event, copyToClipboard) => {
+    copyToClipboard();
+    openTooltip();
+  }, []);
```

Comment:
we don't need useCallback here, its not gonna render over and over again.

```suggestion
  const openTooltip = () => {
    setShowCopyTooltip(true);
    setTimeout(() => setShowCopyTooltip(false), 2000);
  };
  const copyCommand = () => {
    copyToClipboard(command);
    openTooltip();
  };
```